src/main/java/org/spongepowered/common/data/builder/manipulator/mutable/entity/ScreamingDataBuilder.java

```diff
@@ -55,8 +52,8 @@ public ScreamingData create() {
 
     @Override
     public Optional<ScreamingData> createFrom(DataHolder dataHolder) {
-        if (dataHolder.supports(Keys.IS_SCREAMING)) {
-            return Optional.<ScreamingData>of(new SpongeScreamingData(dataHolder.get(Keys.IS_SCREAMING).get()));
+        if (dataHolder instanceof EntityEnderman) {
```

Comment:
Yes, but at the same time, this isn't actually used anywhere since `DataProcessor`s are usually how you'll get the data.